i18n/management/commands/log_i18n_failure.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=missing-docstring
+from i18n.management.utils import log
+
+from django.core.management.base import BaseCommand
+
+class Command(BaseCommand):
+    """
+    Used by the i18n sync to alert that not all steps finished successfully.
+    See bin/sync_i18n.sh for context.
+    """
+    def handle(self, *args, **options):
+        log("WARNING: I18n Sync encountered an error")
```

Comment:
Minor comments mostly out of my own python ignorance:
* Why is the "error" logged as a "WARNING" instead of an "ERROR"? Could everything be okay if there was an error?
* Is there a 'error(...)' or 'warn(...)' function which might be more appropriate?

Reply:
1. No reason at all! I like "ERROR" better, I'll update it.
2. Not right now; the `log` method we're using here is defined at https://github.com/code-dot-org/curriculumbuilder/blob/54f9f52f686eea0909d73c3b7aadb6553f23b949/i18n/management/utils.py#L49-L58 and is pretty minimal. What we really _should_ be using is an actual python logger like we are in https://github.com/code-dot-org/curriculumbuilder/pull/164 but I'm sticking with what we already have for this change